server/lomas_server/constants.py

```diff
@@ -193,7 +193,7 @@ class OpenDPDatasetInputMetric(StrEnum):
 
 # Example pipeline inputs
 OPENDP_PIPELINE = (
-    '{"version": "0.10.0", '
+    '{"version": "0.11.1", '
```

Comment:
maybe get version automatically instead of hardcoding, to avoid having to modify it ourself for each version change